Libraries/Components/AccessibilityInfo/AccessibilityInfo.android.js

```diff
@@ -66,6 +67,15 @@ const AccessibilityInfo = {
     listener.remove();
     _subscriptions.delete(handler);
   },
+
+  /**
+   * Set accessibility focus to a react component.
+   *
+   * See http://facebook.github.io/react-native/docs/accessibilityinfo.html#setaccessibilityfocus
+   */
+  setAccessibilityFocus: function(reactTag: number): void {
+    UIManager.sendAccessibilityEvent(reactTag, 8);
```

Comment:
Instead of set the value 8 here, you should export a new constant in UIManagerModuleConstants.AccessibilityEventTypes:

see: https://github.com/facebook/react-native/blob/26684cf3adf4094eb6c405d345a75bf8c7c0bf88/ReactAndroid/src/main/java/com/facebook/react/uimanager/UIManagerModuleConstants.java#L138

and then use that new constant from here